pdr_backend/util/subgraph_slot.py

```diff
@@ -0,0 +1,201 @@
+from typing import List, Dict, Any, TypedDict
+
+from pdr_backend.util.subgraph import query_subgraph
+from pdr_backend.util.subgraph_predictions import get_subgraph_url
+
+
+class Slot(TypedDict):
```

Comment:
This overlaps heavily with models/slot.py's Slot class. Please resolve.

Reply:
used @dataclass here and created a class named as PredictSlot